src/AspNet.Security.OAuth.Ebay/EbayAuthenticationDefaults.cs

```diff
@@ -0,0 +1,52 @@
+﻿/*
+ * Licensed under the Apache License, Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+ * See https://github.com/aspnet-contrib/AspNet.Security.OAuth.Providers
+ * for more information concerning the license and the contributors participating to this project.
+ */
+
+using Microsoft.AspNetCore.Authentication;
+using Microsoft.AspNetCore.Authentication.OAuth;
+
+namespace AspNet.Security.OAuth.Ebay
+{
+    /// <summary>
+    /// Default values used by the eBay authentication provider.
+    /// </summary>
+    public static class EbayAuthenticationDefaults
+    {
+        /// <summary>
+        /// Default value for <see cref="AuthenticationScheme.Name"/>.
+        /// </summary>
+        public static readonly string AuthenticationScheme = "Ebay";
```

Comment:
Just this one is the odd one out, and _is_ a constant. 😃 

```suggestion
        public const string AuthenticationScheme = "Ebay";
```

Reply:
Ok 😕. There is an issues those explain the reason about that?

Reply:
No, I was just copying what ASP.NET Core's handlers do ([example](https://github.com/dotnet/aspnetcore/blob/main/src/Security/Authentication/Google/src/GoogleDefaults.cs)) when I updated them.

I did explicitly _not_ include this one in the code review suggestion (see adb138479e7af6b5a7a95321a41baccf514b3412).

Reply:
Yes I had seen that, I honestly thought it was an oversight 😁
I agree to follow official developments.